cli/src/main.rs

```diff
@@ -91,10 +100,15 @@ fn main() {
     let config = parse_args();
 
     let vm_name = config.get_name();
-    let lowmem: usize = config.get_mem() * 1024 * 1024;
     let cpus = config.get_cpus();
 
-    let inst = build_instance(vm_name, cpus, lowmem).unwrap();
+    const GB: usize = 1024 * 1024 * 1024;
```

Comment:
Just as an aside, we should constants for things like this squirreled away at a higher level. No action here, just a note for the future.